generate-release/src/github_client.rs

```diff
@@ -114,11 +124,11 @@ impl GithubClient {
         Self { agent, token, repo }
     }
 
-    fn get(&self, path: &str) -> ureq::Request {
+    /// Submits a request to `bevyengine/{repo}`
```

Comment:
```suggestion
    /// Submits a GET request to `bevyengine/{repo}`
```

There are multiple HTTP requests types, such as GET, POST, DELETE, etc. This makes it clear which method is used.